### PR TITLE
F #2095 Introduce `onehost foceupdate` command

### PR DESCRIPTION
**Pull request by xorel**

### CODE REVIEW

File: src/cli/one_helper/onehost_helper.rb [hunk 1]
```diff
@@ -381,6 +381,98 @@ def sync(host_ids, options)
         end
     end
 
+    def forceupdate(host_ids, options)
+        if `id -u`.to_i == 0 || `id -G`.split.collect{|e| e.to_i}.include?(0)
+            STDERR.puts("Cannot run 'onehost sync' as root")
```
Comment (rsmontero) [inline src/cli/one_helper/onehost_helper.rb]: onehost forceupdate ?

File: share/linters/.rubocop.yml [hunk 2]
```diff
@@ -360,7 +360,6 @@ AllCops:
     - src/cli/command_parser.rb
     - src/cli/cli_helper.rb
     - src/cli/one_helper.rb
-    - src/cli/one_helper/onehost_helper.rb
```
Comment (rsmontero) [inline share/linters/.rubocop.yml]: Why is this file being removed from the linter?

File: src/cli/one_helper/onehost_helper.rb [hunk 3]
```diff
@@ -381,6 +381,98 @@ def sync(host_ids, options)
         end
     end
 
+    def forceupdate(host_ids, options)
+        if `id -u`.to_i == 0 || `id -G`.split.collect{|e| e.to_i}.include?(0)
```
Comment (rsmontero) [inline src/cli/one_helper/onehost_helper.rb]: Better use Process.euid / Process.uid

File: src/cli/one_helper/onehost_helper.rb (hunk 1 above)
Comment (xorel) [reply src/cli/one_helper/onehost_helper.rb]: fixed that copy & paste error

File: share/linters/.rubocop.yml (hunk 2 above)
Comment (rsmontero) [reply share/linters/.rubocop.yml]: You are right, my fault

**Pull request merged**